homeassistant/components/simplisafe/alarm_control_panel.py

```diff
@@ -94,6 +98,16 @@ def state(self):
         """Return the state of the entity."""
         return self._state
 
+    @property
+    def supported_features(self) -> int:
```

Comment:
This isn't necessary – it's already provided by the `SimpliSafeEntity` base class.